### DISCUSSION TITLE
Basic Installation through Composer/Docker

### DISCUSSION THREAD

**Discussion by dan-burt** [selectize/selectize.js#1825] [Q&A]
I have a web app published through a Docker container, with an NGinx proxy infront of it to enable SSL encryption.

I would like to add Selectize to my web app so that user entry can be more dynamic.

But how do I install this?

I have tried simply putting the various "install scripts" and most recently looked if it was an NPM package to add to my container deployment (I have a couple already).

I hope to use documentation after this simple step, but nothing I have tried seems to work and the JS Console provides no info to help me troubleshoot.

**Comment by dan-burt** [comment]
I tried to use a `composer` based install command for Selectize.js, which someone picked up on should be an `npm` command.

But I do not know what it should be for an NPM installation. Any guidance would be helpful. There is no other JS-based need currently than to use this component.

I tried using the `cdnjs` script commands but this results in a permission error in the browser's JS console. I expect this to be something to do with using Docker / container? But I do not know, and don't seem to have any further detail in log files as to why...

**Reply by risadams** [reply]
I personally haven't used PHP since roughly 2008, and have never used Composer.  I will need to do some research and try to get a test working.  The package name you have in #1826 seems fine to me.

I will close #1826 in favor of this thread unless we find a deeper bug that needs to be addressed.

I'll let you know once I have an answer.
